### DISCUSSION TITLE
Multiple <CC> commands for ctellraw

### DISCUSSION THREAD

**Discussion by regix1** [Zrips/CMI#9042] [Q&A] [answered]
Hi, 

I was wondering if it's possible to run multiple commands within ![image](https://github.com/user-attachments/assets/e483f9c7-f6f9-4b46-a9fe-4b63711a99a1) ctellraw? I can't seem to figure it out.

I tried two different commands in two different ![image](https://github.com/user-attachments/assets/d21a1e16-c439-4c85-a378-48832d0cca2f) ![image](https://github.com/user-attachments/assets/d561881f-a404-4d39-b043-1c531c64949b) but it only ran the last one and ignored the first one. 

Thanks

**Comment by Zrips** [comment, marked as answer]
This converts to generic JSON type message which only accepts single command per click. Your best work around would be simply to make custom Alias command which contains all of your desired commands and then trigger that one. You can even combine this with specialized commands and have aditional checks and requirements for each of those commands
